app/views/layouts/navbar.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <link rel="stylesheet" href="../../assets/stylesheets/navbar.css">
+  </head>
+  <body>
+
+      <ul>
+        <li class="nav-item">
+          <!-- <a class="navbar-brand" href="path-to-root"><img src="path-to-logo"></a> -->
```

Comment:
You can remove this file as it's a duplication of application.html.erb content

Reply:
Done